### DISCUSSION TITLE
Update DNS records with 3rd party domain provider (with vercel nameserver)

### DISCUSSION THREAD

**Discussion by MrHazimAli** [vercel/community#6109] [Help]
### Summary

Hye, I want I have a domain which bought from 3rd party provider. and last time I already configured using vercel nameserver, changing my plan from pro to hobby cause my domain misconfigured, I need to update the dns but unable to find where, when asking my domain provider, they mentioned that dns is managed by vercel. how do I update my dns?

### Steps to Reproduce

could not find where to update the dns records

**Comment by amyegan** [comment]
Hi @MrHazimAli. You can manage your domains by selecting the "Domains" tab on your Vercel Dashboard.

https://vercel.com/docs/projects/domains/working-with-domains#domain-ownership-and-project-assignment
